README.md

```diff
@@ -51,6 +54,115 @@ infer(network, { 'RAIN': 'T' }).toFixed(4) // 0.2000
 infer(network, { 'RAIN': 'T' }, { 'SPRINKLER': 'F' }).toFixed(4) // 0.2920
 ```
 
+#### inferAll(network: [INetwork](https://github.com/fhelwanger/bayesjs/blob/master/src/types/INetwork.ts), given?: [ICombinations](https://github.com/fhelwanger/bayesjs/blob/master/src/types/ICombinations.ts), options?: [IInferAllOptions](https://github.com/fhelwanger/bayesjs/blob/master/src/types/IInferAllOptions.ts)): [INetworkResult](https://github.com/fhelwanger/bayesjs/blob/master/src/types/INetworkResult.ts))
+Calculate all probabilities from a network and return an object with all results.
+
+This function receives a network, the knowing states, and the options and will return an object with all results. This method will execute the junction tree algorithm on each node's state.
```

Comment:
here you are repeating "return an object with all results." twice, I think this could be more succint like:
```
Calculate all probabilities from a network by receiveing the network, knowing states, and options.
It returns an object with all results.

This method will execute the junction tree algorithm on each node's state.
```

Reply:
also on the tests you use "Node states" but here "node's state", any reason for that?

Reply:
I will fix that.